pkg/apis/settings/settings.go

```diff
@@ -142,6 +147,10 @@ func (s Settings) GetAPIServerName() string {
 	return endpoint.Hostname()
 }
 
+func (*Settings) FromContext(ctx context.Context) coresettings.Injectable {
+	return FromContext(ctx)
+}
```

Comment:
Post-switch to options, this is not used anywhere?

Reply:
Correct. Some methods in `Settings` are left unused due to the references generally changed to `Options`. But still required due to we implementing the interface of `Settings` from core. This is primarily why the coverage drops.

Reply:
Hmm. If that's the case, it is a little unsettling that nothing obvious breaks when I remove it. Does it silently break injection? Maybe we should have a test for this then. Not critical if this goes away, just a bit surprizing.

Reply:
Also we aren't excersising the options switch by default it wont fail it will just using settings. THis is why i was initially pushing to remove in this pr because without we aren't excercising this works for each of the other cases

Reply:
Look at it again, looks like this **method** haven't been used at all. It is also not presented in the interface. 
Let's remove it. Should've been more careful with the public method pruning...

However, what's being used is the **function** of the same name in the same package. That is still being used in injection and merging with `Options.` This explains why removing the earlier doesn't break anything.